src/Illuminate/Database/Query/Builder.php

```diff
@@ -2036,6 +2045,29 @@ public function reorder($column = null, $direction = 'asc')
         return $this;
     }
 
+    /**
+     * Determine if a "limit" or "offset" clause is specified
+     * @param $query
+     * @return bool
+     */
+    protected function isLimitOrOffsetSpecified() : bool {
```

Comment:
remove this type annotation

Reply:
What's wrong with this type annotation? Do we still need to support PHP 5.6?